node/src/actors/chain_manager/handlers.rs

```diff
@@ -617,9 +627,9 @@ impl Handler<PeersBeacons> for ChainManager {
         // Calculate the consensus, or None if there is no consensus
         let consensus_threshold = self.consensus_c as usize;
         let consensus = peers_beacons.consensus(consensus_threshold);
-        let pb = peers_beacons.pb;
+        //let pb = peers_beacons.pb;
         let outbound_limit = peers_beacons.outbound_limit;
-        let pb_len = pb.len();
+        let pb_len = peers_beacons.pb.len();
         let peers_needed_for_consensus = outbound_limit
             .map(|x| {
                 // ceil(x * consensus_threshold / 100)
```

Comment:
Is this line dead too?

Reply:
nope, pb_len is used in subsequent loops